src/server/browser.ts

```diff
@@ -118,6 +119,10 @@ export abstract class Browser extends SdkObject {
     if (this._defaultContext)
       this._defaultContext._browserClosed();
     this.emit(Browser.Events.Disconnected);
+
+    for (const video of this._idToVideo.values())
```

Comment:
Perhaps, emit disconnected event after this?

Reply:
don't see a difference, swapped